dyn/tm/accounts.py

```diff
@@ -435,6 +437,17 @@ def _get(self):
         for key, val in response['data'].items():
             setattr(self, '_' + key, val)
 
+        api_args = {'user_name': self._user_name}
+        response = DynectSession.get_session().execute(self._permission_report_uri, 'POST',
+                                                       api_args)
+
+        for val in response['data']['allowed']:
+            print(val)
+            self.permissions.append(val['name'])
+            for zone in val['zone']:
+                if zone['zone_name'] not in self._zone:
+                    self._zone.append(zone['zone_name'])
+
```

Comment:
This whole block is just here to update the list of permissions after every get right? you might want to make it into its own function and call it on _get. Also, remove the print.